cmd/git/main.go

```diff
@@ -47,6 +49,8 @@ type settings struct {
 	target                 string
 	resultFileCommitSha    string
 	resultFileCommitAuthor string
+	resultErrorMessage     string
+	resultErrorReason      string
```

Comment:
Two thoughts here:
 - If these are coming from the taskrun, how about `taskRunErrorMessage` and `taskRunErrorReason`.
 - Instead of just surfacing the errors, why not surface a generic `taskRunMessage` and `taskRunReason`, maybe an additional `taskRunReasonType` and then you can surface errors or success messages?

Reply:
Yes they are coming from TaskRun. However, the git command itself is not aware of Tekton. From its point of view it is just supplied with two filesystem paths.
If it is not a concern then I can introduce the name changes.

Reply:
For the second thought. Please see https://github.com/shipwright-io/build/pull/901#discussion_r726183874